share/scan/registry.go

```diff
@@ -278,6 +282,15 @@ func (rc *RegClient) GetImageInfo(ctx context.Context, name, tag string, manifes
 
 	imageInfo.RawManifest = body
 
+	if manifestReqType != registry.ManifestRequest_CosignSignature {
+		signatureTag := GetCosignSignatureTagFromDigest(imageInfo.Digest)
+		signatureImageInfo, errCode := rc.GetImageInfo(ctx, name, signatureTag, registry.ManifestRequest_CosignSignature)
+		if errCode != share.ScanErrorCode_ScanErrNone && errCode != share.ScanErrorCode_ScanErrImageNotFound {
+			return nil, errCode
```

Comment:
Does it mean any error for getting signature image info will cause the image not scanned for vulnerability?
I don't think we want that behavior.